charts/nirmata/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 type: application
 name: kyverno
 version: 3.2.9-rc2
-appVersion: v1.12.6-n4k.nirmata.3-rc1
+appVersion: non-rootuser-fix
```

Comment:
@jigar-joshi-nirmata can you pls revert it back to previous tag, this will not affect the FIPS images tag